library/blivet.py

```diff
@@ -559,6 +590,40 @@ def _create(self):
             label = get_format("disklabel", device=self._device.path, label_type=disklabel_type)
             self._blivet.format_device(self._device, label)
 
+class BlivetMDRaidPool(BlivetPool):
+    def create(self):
+        raid_name = "mdraid-%s" % self._pool['name']
+        member_names = self._pool['disks']
```

Comment:
There should already be a method to get the blivet disk instances corresponding to the identifiers in pool['disks'], which can contain any of several identifiers, including UUID=<uuid>, /dev/disk/by-path/<link>, etc. I believe they are accessible as `self._disks`.

Reply:
Fixed: `self._disks` is now obtained as constructor parameter since `BlivetMDRaidPool` no longer inherits from `BlivetPool`